src/lang/id_ID.rc

```diff
@@ -0,0 +1,1583 @@
+    ICON IDI_ARROW, -1, 30, 125, 0, 0
+    COMBOBOX cmb6, 55, 130, 125, 300, CBS_HASSTRINGS | CBS_AUTOHSCROLL | CBS_DROPDOWN | WS_VSCROLL | WS_TABSTOP
+    DEFPUSHBUTTON "OK", IDOK, 75, 160, 50, 14
+    PUSHBUTTON "Batal", IDCANCEL, 130, 160, 50, 14
+    PUSHBUTTON "&Set Ulang", psh1, 5, 160, 55, 14
+}
+
+IDD_DROPDOWNPOPUP DIALOG 0, 0, 120, 150
+CAPTION " "
+STYLE WS_POPUP | WS_DLGFRAME | NOT WS_BORDER
+FONT 9, "MS Shell Dlg"
+{
+    LISTBOX lst1, 0, 0, 120, 150, LBS_WANTKEYBOARDINPUT | LBS_NOINTEGRALHEIGHT | LBS_HASSTRINGS | WS_VSCROLL | WS_TABSTOP, WS_EX_CLIENTEDGE
+}
+
+IDD_DFMSETTINGS DIALOG 0, 0, 210, 135
+CAPTION "Pengaturan Delphi DFM"
+STYLE DS_CENTER | DS_MODALFRAME | WS_POPUPWINDOW | WS_CAPTION
+FONT 9, "MS Shell Dlg"
+{
+    LTEXT "Pengaturan berikut relevan ke DFM:", -1, 35, 5, 171, 20
+    ICON IDI_DFMICON, -1, 5, 5, 0, 0
+    RTEXT "&Codepage:", -1, 6, 32, 70, 12
+    COMBOBOX cmb1, 85, 30, 120, 300, CBS_HASSTRINGS | CBS_AUTOHSCROLL | CBS_DROPDOWN | WS_VSCROLL | WS_TABSTOP
+    AUTOCHECKBOX "&Gunakan komentar raw string", chx1, 5, 50, 200, 14
+    AUTOCHECKBOX "Jangan gunakan &Unicode dan UTF-8", chx2, 5, 70, 200, 14
+    DEFPUSHBUTTON "OK", IDOK, 80, 115, 60, 14
+    PUSHBUTTON "Batal", IDCANCEL, 145, 115, 60, 14
+}
+
+//////////////////////////////////////////////////////////////////////////////
+// RT_VERSION
+
+1 VERSIONINFO
+FILEVERSION     5, 4, 4, 0
+PRODUCTVERSION  5, 4, 4, 0
+FILEOS          0x40004
+FILETYPE        0x1
+{
+    BLOCK "StringFileInfo"
+    {
+        BLOCK "042104B0"
+        {
+            VALUE "CompanyName", "Katayama Hirofumi MZ\0"
+            VALUE "FileDescription", "RisohEditor\0"
+            VALUE "FileVersion", "5.4.4\0"
+            VALUE "LegalCopyright", "Hak cipta (C) 2017-2019 Katayama Hirofumi MZ. Hak cipta dilindungi.\0"
+            VALUE "ProductName", "RisohEditor\0"
+            VALUE "ProductVersion", "5.4.4\0"
+        }
+    }
+    BLOCK "VarFileInfo"
+    {
+        VALUE "Translation", 0x0421, 0x04B0
+    }
+}
+
+//////////////////////////////////////////////////////////////////////////////
+// RT_DLGINIT
+
+IDD_ADDDLGINIT 240
+{
+    cmb2, 0x0143, 0x000D, 0, 0x4243, 0x415F, 0x4444, 0x5453, 0x4952, 0x474E, "\000", 
+    cmb2, 0x0143, 0x000D, 0, 0x424C, 0x415F, 0x4444, 0x5453, 0x4952, 0x474E, "\000", 
+    cmb2, 0x0143, 0x0010, 0, 0x4243, 0x4D45, 0x495F, 0x534E, 0x5245, 0x4954, 0x4554, 0x004D, 
+    0
+}
+
+IDD_MODIFYDLGINIT 240
+{
+    cmb2, 0x0143, 0x000D, 0, 0x4243, 0x415F, 0x4444, 0x5453, 0x4952, 0x474E, "\000", 
+    cmb2, 0x0143, 0x000D, 0, 0x424C, 0x415F, 0x4444, 0x5453, 0x4952, 0x474E, "\000", 
+    cmb2, 0x0143, 0x0010, 0, 0x4243, 0x4D45, 0x495F, 0x534E, 0x5245, 0x4954, 0x4554, 0x004D, 
+    0
+}
+
+//////////////////////////////////////////////////////////////////////////////
+// RISOHTEMPLATE
+
+4 RISOHTEMPLATE "res/1033_RISOHTEMPLATE_4.bin"
+
+5 RISOHTEMPLATE "res/1033_RISOHTEMPLATE_5.bin"
```

Comment:
Can  you add `(lang-id)_RISOHTEMPLATE_?.bin` files?

Reply:
If i followed the code following before.
It must be
`4 RISOHTEMPLATE "res/(lang-id)_RISOHTEMPLATE_4.bin`

`5 RISOHTEMPLATE "res/(lang-id)_RISOHTEMPLATE_5.bin`

`6 RISOHTEMPLATE "res/(lang-id)_RISOHTEMPLATE_6.bin`?
I'll Check to other translation as comparison. Thanks